app/controllers/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new(username: 'Test',
+                     email: 'test@test.com',
+                     password: 'omaromar')
+  end
```

Comment:
- [ ] [required] Simply initialize a user instance by `@user=User.new`

Reply:
[Done] Thanks for the note, if you can give me an idea how to fullfill that **Odin Project** requirement by how to **start with a value already populated**

![2516e570-892a-4bbb-827c-d8cb4229b927](https://user-images.githubusercontent.com/24830039/119822733-468ee680-bef4-11eb-8f35-a16d54828981.png)